src/stan/error_handling/matrix/check_ldlt_factor.hpp

```diff
@@ -28,9 +38,8 @@ namespace stan {
         msg << "is not positive definite. "
             << "last conditional variance is ";
```

Comment:
What does "conditional variance" have to do with positive definiteness?  Is lack of pos defintieness the only reason this could fail?